news/2024/2024-10-06-new-featured-artist-trina-lydia.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: Trina Lydia"
+date: 2024-10-06 18:00:00 +0000
+---
+
+You don't need synesthesia to see why **Trina Lydia**'s music is worth mapping.
+
+![](https://assets.ppy.sh/artists/440/header.jpg)
+
+While **Trina Lydia**'s style is clearly reminiscent of authentic rhythm game tunes, they refuse to stick to a single genre.
+
+Each track that **Trina Lydia** creates is fresh, accompanied by sprinkles of references to other iconic tracks and explorations into new production techniques. It doesn't matter how different each song genre is though — all of **Trina Lydia**'s songs are worth bringing into osu!.
+
+[**Trina Lydia**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/440) is officially home to **8** new songs, including two [originals](/wiki/osu!_originals) produced specifically for community tournaments!
```

Comment:
TETRATONE ALTiSSiMO was for a [dan course](https://osu.ppy.sh/beatmapsets/2234187#mania/4745284), not a tournament